src/utils/streams/concat_stream_providers.test.js

```diff
@@ -61,19 +58,18 @@ describe('concatStreamProviders() helper', () => {
       }),
     ]);
 
-    const errorListener = sinon.stub();
+    const errorListener = jest.fn();
     dest.on('error', errorListener);
 
     try {
       await createPromiseFromStreams([dest]);
```

Comment:
I'm fine with leaving the tests as they are so that the diff between the previous and new tests is smaller, but just incase you didn't know there is a really nice way to test for things like this in jest:

```js
await expect(createPromiseFromStreams([dest])).rejects.toThrowErrorMatchingInlineSnapshot(...)
await expect(errorListener).toMatchInlineSnapshot(...)
```